.circleci/src/workflows/root.yml

```diff
@@ -1,6 +1,12 @@
 jobs:
-  - root-publish-github-release:
+  - root-publish-github-release-client:
       context: Audius Client
       filters:
         branches:
-          only: /(^release-client.*)$/
+          only: /(^release-client-v.*)$/
+  - root-publish-github-release-protocol:
+      # Audius Client context contains the GHI token
+      context: Audius Client
```

Comment:
should this say protocol?

Reply:
Yeah we need the Audius Client context because it has the token we use to create github releases. We could move it potentially